sccp/figures/figureS2.py

```diff
@@ -55,20 +34,20 @@ def makeFigure():
     cmp = 11
 
     _, factors, projs, = openPf2(rank = 39, dataName = 'lupus')
```

Comment:
this is where build is failing in #167 because projs file isn't uploaded (too big for rank 39)